kolena/classification/multiclass/evaluator.py

```diff
@@ -164,15 +169,11 @@ def _compute_test_case_plots(
         or field.name not in ["Precision", "Recall"]  # Omit single-class TC from precision and recall plots
     ]
 
-    confidence_histograms = _compute_confidence_histograms(metrics, confidence_range)
-    if confidence_histograms:
-        plots.extend(confidence_histograms)
-    roc_curve_plot = _compute_test_case_ovr_roc_curve(labels, ground_truths, inferences)
-    if roc_curve_plot:
-        plots.append(roc_curve_plot)
-    confusion_matrix = _compute_test_case_confusion_matrix(test_case_name, ground_truths, metrics)
-    if confusion_matrix:
-        plots.append(confusion_matrix)
+    plots.extend(_compute_confidence_histograms(test_case_name, metrics, confidence_range))
```

Comment:
This will err if there are no confidence histograms:
```
>>> foo = []
>>> foo.extend(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```

If this is the expected usage, we can make `_compute_confidence_histograms` return `[]` instead of `None`

Reply:
Thanks for catching this! I pushed the fix for it.